TestProjects/Playground/Playground.Forms.Droid/Playground.Forms.Droid.csproj

```diff
@@ -10,14 +10,14 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Playground.Forms.Droid</RootNamespace>
     <AssemblyName>Playground.Forms.Droid</AssemblyName>
-    <TargetFrameworkVersion>v8.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v7.1</TargetFrameworkVersion>
```

Comment:
there seems to be some inconsistencies where the csproj has a target framework of 7.1 now but the xamarin forms nuget package targets 8.0

Reply:
yeh but this is consistent with all the other droid projects.